frappe/public/scss/page-builder.scss

```diff
@@ -102,7 +104,7 @@
 
 	&.card-sm {
 		.card-body {
-			padding: 1.5rem;
+			padding: 1.25rem 1.5rem 1.5rem 1.5rem;
```

Comment:
Why unequal padding? If the top padding is more than it is because heading has a larger height.  Did you try fixing that?